specification/web/resource-manager/Microsoft.Web/2016-03-01/Provider.json

```diff
@@ -69,12 +97,200 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/operations": {
+      "get": {
+        "tags": [
+          "Provider"
+        ],
+        "operationId": "Provider_ListOperations",
```

Comment:
This has the same name as https://github.com/Azure/azure-rest-api-specs/pull/1519/files#diff-3adaa26580e5e10147ca02b203c71b4fR46